src/main/kotlin/no/fdk/concept_catalog/service/SkosApNoModelService.kt

```diff
@@ -276,6 +286,72 @@ class SkosApNoModelService(
             }
     }
 
+    private fun addBegrepsRelasjonToConcept(conceptBuilder: ConceptBuilder, concept: Begrep) {
+        concept.begrepsRelasjon
+            ?.let {
+                it.forEach { it ->
+                    val relasjon = it.relasjon
+                    val relatertBegrep = it.relatertBegrep
+
+                    if (relasjon == ASSOCIATIVE) {
+                        val associativeRelationBuilder = conceptBuilder.associativeRelationBuilder()
+
+                        it.beskrivelse
+                            ?.filterValues { it.toString().isNotBlank() }
```

Comment:
Ikke bruk samme variabelnavn for scope-funksjoner inne i andre scope-funksjoner:
```suggestion
                            ?.filterValues { beskrivelse -> beskrivelse.toString().isNotBlank() }
```